pmpro-stripe-billing-limits.php

```diff
@@ -32,16 +33,17 @@ function pmprosbl_pmpro_after_checkout($user_id)
 	//billing limit? gateway stripe?
 	if($pmpro_level->billing_limit > 0 && $order->gateway == "stripe")
 	{
-		//customer id is on order
-		$customer_id = $order->subscription_transaction_id;
+	
+	  //subscription id is on the order
+		$subscription_id = $order->subscription_transaction_id;
 		
-		//get sub id via Stripe API
-		$subscription = pmprosbl_getStripeSubscription($customer_id, $order->code);
+		//customer id is in user meta
+		$customer_id = get_user_meta($user_id, "pmpro_stripe_customerid", true);

```

Comment:
This seems like a very good change. In fact, I don't understand how the plugin could be working at all without it. @LMNTL are you using your branch (that contains this fix) in production? I'm thinking of just using it, too, since there hasn't been much activity here for quite awhile.